fractal-puppy/Engine
# All skills info added

Now, all the skills have description, name, cd, damage and mana cost.

To test:
- Play the game and open the skilltree.
- Check that all the skills (locked and unlocked) show related info. Name, description, icon, cooldown, damage and mana cost.
